horace_core/sqw/@sqw/mask_pixels.m

```diff
@@ -1,24 +1,24 @@
-function wout = mask_pixels (win, mask_array)
+function wout = mask_pixels (win, keep_info)
```

Comment:
You now have a confusing mix of
`keep_array`
`keep_info`
`keep_obj`
scattered through the code. If I had to, I would pick `keep_array` but still prefer `mask_array`

Reply:
`mask_array` assumes that `true` masks correspondent value while `keep_array` -- otherwise -- as it is in the code. 
`keep_array` is used where it is array. Here it may be `PixelData` object (according to description) Need to write test for that and may be fix, but this is another ticket.